between random: 7651216
between random: 7651211
between random: 7651216
between random: 49507
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
timeout: the monitored command dumped core
Aborted
output directory: temp/21
implementation directory: /home/chris/TVOQE_UPGRADE_27/quic/quic-go/server/
implementation command: ./server -p 4443 127.0.0.1
../quic_server_test_newconnectionid_error (0) ...
implementation command: ['./server', '-p', '4443', '127.0.0.1']
quic_process pid: 6552
timeout 100 ./build/quic_server_test_newconnectionid_error seed=112 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 134
FAIL
error: 1 tests(s) failed
